README.md

```diff
@@ -134,23 +134,70 @@ interface "wasi:kv/data/set-many" {
   // Notice that this interface is non-atomic, meaning that the key-value pairs
   // may not be consistent.
   use { Error, key } from "wasi:kv/types"
-  
+
   set-many: func(key_values: stream<(key, stream<u8>)>) -> result<_, Error>
 }
 
+interface "wasi:kv/data/range" {
+  // request using a range of keys
+  use { Error, payload, key-range, key-value } from "wasi:kv/types"
+
+  // returns keys within the range
+  get-keys: func(range: key-range) -> result<stream<key>, Error>
+
+  // returns key-value tuples within the key range
+  get-range: func(range: key-range) -> Result<stream<key-value>, Error>
+
+  // returns the number of keys that match the range
+  count-range: func(range: key-range) -> result<u64, Error>
+
+  // deletes keys that match the range
+  delete-range: func(range: key-range) -> result<_, Error>
+}
+
 interface "wasi:kv/types" {
-  resource Error { 
+  resource Error {
```

Comment:
I think this has to be `error` instead of `Error`.

Reply:
fixed